contactformupdate

```diff
@@ -0,0 +1,228 @@
+<?php
+
+// check for form submission
+if (isset($_POST['submit'])) {
+
+    // inputs from browser
+	$name     = trim($_POST['name']);
+	$phone    = trim($_POST['phone']);
+	$email    = trim($_POST['email']);
+	$dropdown = trim($_POST['dropdown']);
+	$message  = trim($_POST['message']);
+
+    // email recipient as subject
+	$to      = "example@rawad.com";
+	$subject = "New Message";
+
+    // validate inputs
+	$errors = validateInputs($email, $phone, $name, $message);
+
+	if (empty($errors)) {
+        // no errors, send the mail
+		if (mail($to, $subject, $message, "From: $name <$email>")); {
+			return true;
+		}
+		return 'Message failed to send.';
+	} else {
+        // validation errors
+		return implode("<br />", $errors);
```

Comment:
Change this line from `return implode("<br />"), $errors` to `$strValidationMessage = implode("<br />", $errors);`